src/stcal/ramp_fitting/ols_fit.py

```diff
@@ -658,6 +661,170 @@ def ols_ramp_fit_single(ramp_data, buffsize, save_opt, readnoise_2d, gain_2d, we
     opt_info : tuple
         The tuple of computed optional results arrays for fitting.
     """
+    # use_c = False
+    # use_c = True
+    use_c = ramp_data.dbg_run_c_code
+    if use_c:
+        print(" ")
+        print("=" * 80)
+        c_start = time.time()
```

Comment:
Alternatively, instead of completely removing these lines, they could be turned into log.debug() statements, so that folks could still do timing comparisons.